.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+	<extension>
+		<groupId>org.eclipse.tycho</groupId>
```

Comment:
I think this should be done in the main pom using the plugin management section. This is basically in a hidden file with important impact on the build. So it would be better to be out front in the base pom, no?

Reply:
This is unfortunately not possible because Maven-extensions are considered very early in the start up of a Maven build and contribute to the Core classloader: https://maven.apache.org/guides/mini/guide-using-extensions.html

The main (actually the only) task performed by the Tycho-build plug-in is to generate the polyglot pom-files for Plug-ins, Features, etc. that are already consumed when the regular pom files are processed. Therefore the extension cannot be defined in the main pom.xml (but would definitely better to do so).
You will find such file in every Eclipse project that uses Tycho-pomless.

Reply:
It works quite well in the Transformer build.

https://github.com/eclipse/transformer/blob/cfc100bd10c3b2d2360bd5617b4e32bbb54e3dcf/org.eclipse.transformer.parent/pom.xml#L97

Reply:
Perhaps tycho-build does more than lifecycle participant.

Reply:
> Perhaps tycho-build does more than lifecycle participant.

Yes it does more.
Tycho-build (previously named Tycho-'pomless') generates, during the startup of the Maven-build, the 'polyglot'-pom-files for those projects that don't have a regular pom-file. That's the reason many pom.xml files can be removed from git with this PR. 

For the transformer project this is probably not required since you have explicit regular pom.xml files.
The downside of this approach is that you have to keep the data in the MANIFEST.MF/feature.xml etc. and pom.xml in sync manually (which the version most of the time). While when you use Tycho-pomless you only manage e.g. the MANIFEST.MF and a corresponding ployglot-pom is generated during the start-up of the build.

Reply:
As per other comments. This can't be done in parent pom.